basic-sql-operations.md

```diff
@@ -178,25 +150,27 @@ To show all the indexes in a table, use the `SHOW INDEX` statement:
 SHOW INDEX from person;
 ```
 
-### Delete an index
-
-To delete an index, use the `DROP INDEX` or `ALTER TABLE` statement:
+To delete an index, use the `DROP INDEX` or `ALTER TABLE` statement. `DROP INDEX` can embed `ALTER TABLE` statements.:
 
 {{< copyable "sql" >}}
 
 ```sql
-DROP INDEX person_num ON person;
+DROP INDEX person_id ON person;
 ```
 
 {{< copyable "sql" >}}
 
 ```sql
-ALTER TABLE person DROP INDEX person_num;
+ALTER TABLE person DROP INDEX person_unique_id;
 ```
 
-## Insert, select, update, and delete data
+> **Note:**
+> 
+> DDL operations are not transactions. You don't need corresponding COMMIT statements when executing DDL.
+>
+> Common DML features are adding, modifying, and deleting tables. The corresponding commands are CREATE,  ALTER,and DELETE.
```

Comment:
```suggestion
> Common DML features are adding, modifying, and deleting table records. The corresponding commands are `INSERT`, `UPDATE`, and `DELETE`.
```
btw, I think this sentence should not be included in the **Note**. What do you think?